cpu/atmega_common/periph/spi.c

```diff
@@ -92,7 +92,7 @@ void spi_release(spi_t bus)
 void spi_transfer_bytes(spi_t bus, spi_cs_t cs, bool cont,
                         const void *out, void *in, size_t len)
 {
-    uint8_t *out_buf = (uint8_t *)out;
+    const uint8_t *out_buf = (const uint8_t *)out;
```

Comment:
The *cast* isn't necessary (void pointers are cast implicitly on assignment). The *type* should still stay `const uint8_t *`.